### PR TITLE
2/3 Verify signer

### PR DESCRIPTION
**Pull request by neotheprogramist**
fixes https://github.com/neotheprogramist/cairo-webauthn/issues/2
based on https://github.com/cartridge-gg/cairo-webauthn/pull/26

### CODE REVIEW

**Review by neotheprogramist — COMMENTED** [review]
Can we remove that?

File: account_sdk/src/tests/webauthn_signer_test.rs [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            },
+            BlockId::Tag(BlockTag::Latest),
+        )
+        .await
+        .expect("failed to call contract");
+
+    assert_eq!(result.len(), 1);
+    assert_eq!(result[0], felt!("1"));
+
+    // let result = new_account.execute(vec![Call {
```
Comment (neotheprogramist) [inline account_sdk/src/tests/webauthn_signer_test.rs:76]: Can we remove that?

File: cartridge_account/src/interface.cairo [hunk 2]
```diff
@@ -50,4 +50,18 @@ trait AccountCamelABI<TState> {
     fn getPublicKey(self: @TState) -> felt252;
     fn isValidSignature(self: @TState, hash: felt252, signature: Array<felt252>) -> felt252;
     fn supportsInterface(self: @TState, interfaceId: felt252) -> bool;
+    fn verifyWebauthnSigner(
+        ref self: TState, 
+        pub_x: u256,
+        pub_y: u256, // public key as point on elliptic curve
+        // r: u256, // 'r' part from ecdsa
```
Comment (neotheprogramist) [inline cartridge_account/src/interface.cairo:57]: Should'n that be uncommented?

Comment (ThePinion) [reply cartridge_account/src/interface.cairo:57]: Yes, probably. As a side note, what is the point of the interface if everything works even though the interface doesn't match the implementation?

File: account_sdk/src/tests/webauthn_signer_test.rs (hunk 1 above)
Comment (ThePinion) [reply account_sdk/src/tests/webauthn_signer_test.rs]: Removed

File: cartridge_account/src/interface.cairo (hunk 2 above)
Comment (ThePinion) [reply cartridge_account/src/interface.cairo]: I've uncommented this.

**Pull request closed without merging**